### PR TITLE
[FR] Add `Client/Release_Stream/Lazer/File_Storage`

### PR DESCRIPTION
**Pull request by ItsShamed**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Client/Release_stream/Lazer/File_storage/fr.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# Stockage de fichiers dans osu!(lazer)
+
+Par défaut, osu!(lazer) stocke les fichiers de beatmaps, de skins, et de replays de scores dans les répertoires suivants :
+
+- `%appdata%/osu/files` (Windows),
+- `~/.local/share/osu/files` (Linux),
+- `~/Library/Application Support/osu/files` (macOS),
+- `Android/data/sh.ppy.osulazer/files/files` (Android).
+
+Sur les plateformes PC, le stockage de fichiers peut aussi être bougé dans son entièreté vers une autre emplacement, en appuyant sur le bouton `Changer l'emplacement du dossier...`
```
Comment (Pisapou) [inline wiki/Client/Release_stream/Lazer/File_storage/fr.md:10]:
```suggestion
Sur les ordinateurs, le stockage de fichiers peut aussi être bougé dans son entièreté vers une autre emplacement, en appuyant sur le bouton `Changer l'emplacement du dossier...` dans les paramètres du client.
```

File: wiki/Client/Release_stream/Lazer/File_storage/fr.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+va être stocké à l'emplacement
+
+```
+files/1/1a/1a47929b6056d34d25a95eeb2012395ceed66af6f40cc37c898a08482d6325d2
+```
+
+Cela permet d'économiser de l'espace de stockage en évitant les duplications du même fichier sur le disque, et empêche les utilisateurs (ou autres applications) d'altérer des fichiers qui ne devraient pas l'être. Tous les inconvénients qui découlent de ce choix par rapport à stable ont été résolus soit en autorisant l'exportation, soit en ajoutant des fonctionnalités de gestion en jeu, ou seront abordés en temps voulu avec de nouvelles fonctionnalités.
+
+Pour le moment, la procédure correcte pour manuellement modifier les skins ou les beatmaps en dehors des fonctions disponibles en jeu est d'exporter l'élément en question, le changer, et ensuite le réimporter de nouveau avec les changements désirés.
```
Comment (Pisapou) [inline wiki/Client/Release_stream/Lazer/File_storage/fr.md:30]:
```suggestion
Pour le moment, la procédure correcte pour manuellement modifier les skins ou les beatmaps en dehors des fonctions disponibles en jeu est d'exporter l'élément en question, les changer, et ensuite les réimporter de nouveau avec les changements désirés.
```

File: wiki/Client/Release_stream/Lazer/File_storage/fr.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+## Migration depuis osu!(stable)
+
+### Avec des liens physiques
```
Comment (Pisapou) [inline wiki/Client/Release_stream/Lazer/File_storage/fr.md:34]:
```suggestion
### Par des liens physiques
```

**Review by Pisapou — CHANGES_REQUESTED** [review]

**Comment by Pisapou** [comment]
Par ailleurs, n'hésite pas à rejoindre la serveur discord osu! French (https://discord.com/invite/vfrHNQZ) si tu comptes t'impliquer dans la traduction du wiki!

File: wiki/Client/Release_stream/Lazer/File_storage/fr.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+Un lien physique est conceptuellement semblable à un *raccourci* dû au fait que c'est une méthode qui permet à un utilisateur d'avoir le même fichier à différents emplacements sur leur système de fichiers. Cependant, alors que les raccourcis ne sont juste que des fichiers pleins qui pointent vers un fichier différent (et requièrent ainsi un espace additionnel), les liens physiques fonctionnent un niveau plus profond, au niveau du système de fichiers en lui-même.
+
+Pour faire simple, deux fichiers liés physiquement sont simplement deux noms différents pour le même fragment d'espace sur le disque dur. Cela veut dire qu'une fois que la migration se termine via des liens physiques, les répertoires `Songs` (sur stable) et `files` (sur lazer) vont tous les deux contenir des fichiers qui pointent vers des données partagées sur le disque. Supprimer des fichiers sur une installation ne va pas affecter l'autre. Changer les fichiers entraînera la séparation des fichiers, et ne vont alors pas non plus affecter l'autre installation, tant qu'osu! est habitué à faire ces changements.
```
Comment (Pisapou) [inline wiki/Client/Release_stream/Lazer/File_storage/fr.md:40]:
```suggestion
Pour faire simple, deux fichiers liés physiquement sont simplement deux noms différents pour le même fragment d'espace sur le disque dur. Cela veut dire qu'une fois que la migration se termine via des liens physiques, les répertoires `Songs` (sur stable) et `files` (sur lazer) vont tous les deux contenir des fichiers qui pointent vers des données partagées sur le disque. Supprimer des fichiers sur une installation ne va pas affecter l'autre. Changer les fichiers entraînera la séparation des fichiers, et n'affectera donc pas non plus l'autre installation, tant qu'osu! est habitué à faire ces changements.
```

File: wiki/Client/Release_stream/Lazer/File_storage/fr.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+À noter que si l'utilisation du disque est vérifié depuis la fenêtre `Propriétés` dans l'Explorateur ou quelconque autre méthode similaire, il semblerait que les dossiers `Songs` et `files` utilisent de l'espace disque, ce qui pourrait induire en erreur les utilisateurs à penser que les données sont stockées deux fois. C'est un artéfact de la détermination de taille, et les vraies données stockées sur le disque ne sont **pas** dupliquées. Cela peut être vérifié en comparant la valeur totale de l'espace disponible sur le disque entier avant et après la migration.
```
Comment (Pisapou) [inline wiki/Client/Release_stream/Lazer/File_storage/fr.md:42]:
```suggestion
À noter que si l'utilisation du disque est vérifiée depuis la fenêtre `Propriétés` dans l'Explorateur ou d'une quelconque autre méthode similaire, il semblerait que les dossiers `Songs` et `files` utilisent de l'espace disque, ce qui pourrait induire en erreur les utilisateurs à penser que les données sont stockées deux fois. C'est un artéfact de la détermination de taille, et les vraies données stockées sur le disque ne sont **pas** dupliquées. Cela peut être vérifié en comparant la valeur totale de l'espace disponible sur le disque entier avant et après la migration.
```

**Review by Pisapou — CHANGES_REQUESTED** [review]

File: wiki/Client/Release_stream/Lazer/File_storage/fr.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
+# Stockage de fichiers dans osu!(lazer)
+
+Par défaut, osu!(lazer) stocke les fichiers de beatmaps, de skins, et de replays de scores dans les répertoires suivants :
+
+- `%appdata%/osu/files` (Windows),
+- `~/.local/share/osu/files` (Linux),
+- `~/Library/Application Support/osu/files` (macOS),
+- `Android/data/sh.ppy.osulazer/files/files` (Android).
+
+Sur les ordinateurs, le stockage de fichiers peut aussi être bougé dans son entièreté vers une autre emplacement, en appuyant sur le bouton `Changer l'emplacement du dossier...` dans les paramètres du client.
```
Comment (Ganondorf-1) [inline wiki/Client/Release_stream/Lazer/File_storage/fr.md]:
```suggestion
Sur les ordinateurs, le stockage de fichiers peut aussi être déplacé dans son entièreté vers un autre emplacement, en appuyant sur le bouton `Changer l'emplacement du dossier...` dans les paramètres du client.
```

File: wiki/Client/Release_stream/Lazer/File_storage/fr.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+va être stocké à l'emplacement
+
+```
+files/1/1a/1a47929b6056d34d25a95eeb2012395ceed66af6f40cc37c898a08482d6325d2
+```
+
+Cela permet d'économiser de l'espace de stockage en évitant les duplications du même fichier sur le disque, et empêche les utilisateurs (ou autres applications) d'altérer des fichiers qui ne devraient pas l'être. Tous les inconvénients qui découlent de ce choix par rapport à stable ont été résolus soit en autorisant l'exportation, soit en ajoutant des fonctionnalités de gestion en jeu, ou seront abordés en temps voulu avec de nouvelles fonctionnalités.
+
+Pour le moment, la procédure correcte pour manuellement modifier les skins ou les beatmaps en dehors des fonctions disponibles en jeu est d'exporter l'élément en question, le changer, et ensuite le réimporter de nouveau avec les changements désirés.
```
Comment (Ganondorf-1) [inline wiki/Client/Release_stream/Lazer/File_storage/fr.md:30]:
```suggestion
Pour le moment, la procédure correcte pour manuellement modifier les skins ou les beatmaps en dehors des fonctions disponibles en jeu est d'exporter l'élément en question, le changer, et ensuite le réimporter avec les changements désirés.
```

File: wiki/Client/Release_stream/Lazer/File_storage/fr.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+
+## Migration depuis osu!(stable)
+
+### Par des liens physiques
+
+Sur la plupart des systèmes Windows, osu!(lazer) va être capable d'importer des données depuis la version stable du jeu sans avoir à créer de secondes copies des données sur le disque. Cela est possible grâce à une fonctionnalité du système d'exploitation appelée *"liens physiques"*.
+
+Un lien physique est conceptuellement semblable à un *raccourci* dû au fait que c'est une méthode qui permet à un utilisateur d'avoir le même fichier à différents emplacements sur leur système de fichiers. Cependant, alors que les raccourcis ne sont juste que des fichiers pleins qui pointent vers un fichier différent (et requièrent ainsi un espace additionnel), les liens physiques fonctionnent un niveau plus profond, au niveau du système de fichiers en lui-même.
```
Comment (Ganondorf-1) [inline wiki/Client/Release_stream/Lazer/File_storage/fr.md:38]:
```suggestion
Un lien physique est conceptuellement semblable à un *raccourci* dû au fait que c'est une méthode qui permet à un utilisateur d'avoir le même fichier à différents emplacements sur son système de fichiers. Cependant, alors que les raccourcis ne sont que de simples fichiers qui pointent vers un fichier différent (et requièrent ainsi un espace additionnel), les liens physiques fonctionnent à un niveau plus profond, au niveau du système de fichiers en lui-même.
```

File: wiki/Client/Release_stream/Lazer/File_storage/fr.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+
+Pour faire simple, deux fichiers liés physiquement sont simplement deux noms différents pour le même fragment d'espace sur le disque dur. Cela veut dire qu'une fois que la migration se termine via des liens physiques, les répertoires `Songs` (sur stable) et `files` (sur lazer) vont tous les deux contenir des fichiers qui pointent vers des données partagées sur le disque. Supprimer des fichiers sur une installation ne va pas affecter l'autre. Changer les fichiers entraînera la séparation des fichiers, et n'affectera donc pas non plus l'autre installation, tant qu'osu! est habitué à faire ces changements.
+
+À noter que si l'utilisation du disque est vérifiée depuis la fenêtre `Propriétés` dans l'Explorateur ou d'une quelconque autre méthode similaire, il semblerait que les dossiers `Songs` et `files` utilisent de l'espace disque, ce qui pourrait induire en erreur les utilisateurs à penser que les données sont stockées deux fois. C'est un artéfact de la détermination de taille, et les vraies données stockées sur le disque ne sont **pas** dupliquées. Cela peut être vérifié en comparant la valeur totale de l'espace disponible sur le disque entier avant et après la migration.
```
Comment (Ganondorf-1) [inline wiki/Client/Release_stream/Lazer/File_storage/fr.md]:
```suggestion
À noter que si l'utilisation du disque est vérifiée depuis la fenêtre `Propriétés` dans l'Explorateur ou d'une quelconque autre méthode similaire, il semblera que les dossiers `Songs` et `files` utilisent de l'espace disque, ce qui pourrait induire en erreur les utilisateurs à penser que les données sont stockées deux fois. C'est un artéfact de la détermination de taille de fichier, et les vraies données stockées sur le disque ne sont **pas** dupliquées. Cela peut être vérifié en comparant la valeur totale de l'espace disponible sur le disque entier avant et après la migration.
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**